scale-testing/README.md

```diff
@@ -0,0 +1,18 @@
+#ChRIS Performance and Scale Testing
+
+Scripts to test the scalability, durability, and overall performance of the ChRIS system
+
+## Setup:
+### Prerequisites:
+* a running pman and pfioh, either locally or on the MOC
+### config.cfg:
+* PLUGIN: the image plugin to be tested
+* SWIFT_PATH: the path to the swift-credentials.cfg file (only necessary if using swift storage)
+* CHRIS_PATH: the path to the ChRIS-E2E directory
+* PMAN_IP: the IP address on which pman is running (sudo oc get route pman)
+* PFIOH_IP: the IP address on which pman is running (sudo oc get route pfioh)
+* SIZE:
+* OP:
+
+## test_local.py
+### 
```

Comment:
@chloenayon Can you update this to also include the steps to start this testing ? Like :

- How to set up the local env (which script to execute)
- Update config.cfg 
- How to run tests
- Where are the results stored

Reply:
@chloenayon From README it is still not clear which file to run. For e.g. I ran `test_time.py` and `test_request.py`, but i saw nothing happening. Add info like 
`python3 test_local.py`